tests/governance/test_virtual_keys_access.py

```
@@ -0,0 +1,479 @@
+        provider1_found = any(
+            m["id"].startswith(f"{provider1}/") for m in filtered_models
+        )
+        provider2_found = any(
+            m["id"].startswith(f"{provider2}/") for m in filtered_models
+        )
+        assert provider1_found
+        assert provider2_found
+
+    @pytest.mark.virtual_keys
+    @pytest.mark.integration
+    def test_list_models_without_vk_header_returns_all(self, governance_client):
```

Ruff (0.14.5)

Unused method argument: `governance_client`

(ARG002)